src/sections/Welcome.js

```diff
@@ -0,0 +1,109 @@
+import { css } from "@emotion/core"
+import styled from "@emotion/styled"
+import React from "react"
+import Particles from "react-tsparticles"
+import Button from "../components/Button"
+
+function Welcome(props) {
+  const ParticlesContainer = styled.div`
+    width: 100vw;
+    height: 90vh;
+    position: relative;
+    display: flex;
+    justify-content: center;
+  `
+  const WelcmeContainer = styled.div`
+    position: absolute;
+    z-index: 2;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-color: transparent;
+
+    top: 20%;
+
+    & > h1 {
+      text-align: center;
+      margin-bottom: 5vh;
+      font-size: calc(2.8rem + 1vw);
+    }
+  `
+  return (
+    <ParticlesContainer>
+      <WelcmeContainer>
+        <h1>CodeProjects @ GISNY </h1>
+        <Button> Activities</Button>
+      </WelcmeContainer>
+      <Particles
```

Comment:
I would outsource the Particles into an own component, because it is a lot of code 😄